src/main/java/practice/StreamPractice.java

```diff
@@ -59,7 +87,11 @@ public List<Person> getWorkablePeople(int fromAge, int femaleToAge,
      * return the names of all cats whose owners are women from `femaleAge` years old inclusively.
      */
     public List<String> getCatsNames(List<Person> peopleList, int femaleAge) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(p -> p.getSex().equals(Person.Sex.WOMAN) && p.getAge() >= femaleAge)
```

Comment:
```suggestion
                .filter(p -> Person.Sex.WOMAN.equals(p.getSex()) && p.getAge() >= femaleAge)
```
It would be more NPE safety.